src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseSign1Message.cs

```diff
@@ -525,6 +525,43 @@ public bool VerifyDetached(AsymmetricAlgorithm key, byte[] detachedContent, byte
             return VerifyCore(coseKey, detachedContent, null, associatedData);
         }
 
+        /// <summary>
+        /// Verifies that the signature is valid for the content using the specified key.
+        /// </summary>
+        /// <param name="key">The public key that is associated with the private key that was used to sign the content.</param>
+        /// <param name="detachedContent">The content that was previously signed.</param>
+        /// <param name="associatedData">The extra data associated with the signature, which must match the value provided during signing.</param>
+        /// <returns><see langword="true"/> if the signature is valid; otherwise, <see langword="false"/>.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="key"/> or <paramref name="detachedContent"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException"><paramref name="key"/> is of an unsupported type.</exception>
```

Comment:
Is there a path that makes an ArgumentException for an unsupported kind of CoseKey?  I feel like that's just leftover from the AsymmetricAlgorithm overloads.

Since the other overloads already have it, go ahead and leave this here; but if the question inspires followup... cool :smile: